xtext-gradle-plugin/src/integTest/java/org/xtext/gradle/test/WhenSettingsChange.xtend

```diff
@@ -0,0 +1,53 @@
+package org.xtext.gradle.test
+
+import org.junit.Ignore
+import org.junit.Test
+
+class WhenSettingsChange extends AbstractIntegrationTest {
+
+	override setup() {
+		super.setup
+		buildFile << '''
+			apply plugin: 'org.xtext.xtend'
+			
+			dependencies {
+				compile 'org.eclipse.xtend:org.eclipse.xtend.lib:2.9.0-SNAPSHOT'
+			}
+		'''
+	}
+
+	@Test @Ignore
+	def void shouldRecompileWhenSettingsChange() {
+		// given
+		createFile('src/main/java/HelloWorld.xtend', '''
+			class HelloWorld {
+				
+				def void helloWorld() {
+					#['hello', 'world'].forEach[println(toFirstUpper)]
+				}		
+				
+			}
+		''')
+		buildFile << '''
+			xtext {
```

Comment:
We cane save a few lines here with xtext.languages.xtend.generator.javaSourceLevel